docs/sources/administration/roles-and-permissions/access-control/rbac-fixed-basic-role-definitions/index.md

```diff
@@ -127,38 +127,8 @@ This feature is behind the `accessControlOnCall` feature toggle.
-| `plugins:grafana-oncall-app:user-settings-editor`         | `plugins.app:access`<br>`grafana-oncall-app.user-settings:read`<br>`grafana-oncall-app.user-settings:write`                                                                                                                                                                                                                                                                                                                                                           | Read and update user's own OnCall user settings.                                                                                  |
-| `plugins:grafana-oncall-app:user-settings-admin`          | `plugins.app:access`<br>`grafana-oncall-app.user-settings:read`<br>`grafana-oncall-app.user-settings:write`<br>`grafana-oncall-app.user-settings:admin`                                                                                                                                                                                                                                                                                                               | Read and update OnCall user settings for all users.                                                                               |
-| `plugins:grafana-oncall-app:settings-reader`              | `plugins.app:access`<br>`grafana-oncall-app.other-settings:read`                                                                                                                                                                                                                                                                                                                                                                                                      | Read OnCall settings.                                                                                                             |
-| `plugins:grafana-oncall-app:settings-editor`              | `plugins.app:access`<br>`grafana-oncall-app.other-settings:read`<br>`grafana-oncall-app.other-settings:write`                                                                                                                                                                                                                                                                                                                                                         | Read and update OnCall settings.                                                                                                  |
+If you are using [Grafana OnCall](https://grafana.com/docs/oncall/latest/get-started/), you can try out the integration between Grafana OnCall and RBAC. For a detailed list of the available OnCall RBAC roles, see the table in the Grafana
+OnCall documentation [here](https://grafana.com/docs/oncall/latest/user-and-team-management/#available-grafana-oncall-rbac-roles--granted-actions).
```

Comment:
- Semantic line breaks for better line based diffing in the GitHub UI in the future. (https://sembr.org)
- Prefer "refer to" over "see" (https://github.com/grafana/writers-toolkit/blob/main/vale/Grafana/ReferTo.yml#L3)
- Avoid generic reference "here" (Available Grafana OnCall RBAC roles + granted actions)

```suggestion
If you are using [Grafana OnCall](https://grafana.com/docs/oncall/latest/get-started/), you can try out the integration between Grafana OnCall and RBAC.
For a detailed list of the available OnCall RBAC roles, refer the table in [Available Grafana OnCall RBAC roles and granted actions](https://grafana.com/docs/oncall/latest/user-and-team-management/#available-grafana-oncall-rbac-roles--granted-actions).
```

Reply:
thanks for the tips @jdbaldry! 🙏

Reply:
No problemo! I'll get the Vale linter to do this automatically for us soon :)

Thanks for incorporating the feedback!